webapp/src/components/App/Studies/CreateStudyDialog.tsx

```diff
@@ -0,0 +1,162 @@
+import debug from "debug";
+import { useSnackbar } from "notistack";
+import { useTranslation } from "react-i18next";
+import { AxiosError } from "axios";
+import { usePromise } from "react-use";
+import * as R from "ramda";
+import { GenericInfo, StudyPublicMode } from "../../../common/types";
+import useEnqueueErrorSnackbar from "../../../hooks/useEnqueueErrorSnackbar";
+import { createStudy } from "../../../redux/ducks/studies";
+import { getStudyVersionsFormatted, getGroups } from "../../../redux/selectors";
+import useAppSelector from "../../../redux/hooks/useAppSelector";
+import useAppDispatch from "../../../redux/hooks/useAppDispatch";
+import FormDialog from "../../common/dialogs/FormDialog";
+import StringFE from "../../common/fieldEditors/StringFE";
+import SelectFE from "../../common/fieldEditors/SelectFE";
+import Fieldset from "../../common/Fieldset";
+import CheckboxesTagsFE from "../../common/fieldEditors/CheckboxesTagsFE";
+import { SubmitHandlerPlus } from "../../common/Form";
+
+const logErr = debug("antares:createstudyform:error");
+
+interface FieldValues {
+  name: string;
+  version: string;
+  publicMode: StudyPublicMode;
+  groups: string[];
+  tags: string[];
+}
+
+interface Props {
+  open: boolean;
+  onClose: VoidFunction;
+}
+
+function CreateStudyDialog(props: Props) {
+  const [t] = useTranslation();
+  const { open, onClose } = props;
+  const { enqueueSnackbar } = useSnackbar();
+  const enqueueErrorSnackbar = useEnqueueErrorSnackbar();
+  const versionList = useAppSelector(getStudyVersionsFormatted);
+  const mounted = usePromise();
+  const dispatch = useAppDispatch();
+
+  const groupList = useAppSelector(getGroups);
```

Comment:
the groups in the store are wrong.
Those stored are the one with details (containing the list of users associated with the group) which are limited to those which the user is admin of (or superadmin).
Previous `getGroups()` call listed all the groups the user is in.
Either make two calls to merge the two lists (with and without details) in the store or create a separate store.
Note that the `getGroups` with detail is only needed when the user go to the settings page (where he needs to have the list of users associated with groups so to modifiy them if admin)